.github/ISSUE_TEMPLATE/onboarding_to_cluster.md

```diff
@@ -75,7 +75,19 @@ assignees: ""
 
     LIST OF USERS:
 
-6. **GPG Key** (OPTIONAL)
+6. ** Namespace Quota **
+
+    <!--
+    Please pick the resource tier you need for your project from
+    this list: https://github.com/operate-first/support/blob/main/docs/quotas.md
+
+    If none of the tiers are applicable, please write "CUSTOM" and
```

Comment:
Please add something like:

> Please don't be afraid to choose rather smaller tear first. Upgrading quota is easy, in case you start hitting usage limits. And we will gladly upgrade you. Quota is here to prevent over-reservation (resource requests) not actual usage.

Reply:
Updated